eras/conway/impl/src/Cardano/Ledger/Conway/Rules/Utxow.hs

```diff
@@ -5,45 +5,173 @@
 {-# LANGUAGE FlexibleInstances #-}
 {-# LANGUAGE MultiParamTypeClasses #-}
 {-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeApplications #-}
 {-# LANGUAGE TypeFamilies #-}
-{-# LANGUAGE TypeOperators #-}
```

Comment:
`TypeOperators` extension is needed for ghc-9.6 compatibility. You'll see that CI will fail for ghc-9.6.1